README.md

```diff
@@ -454,6 +461,51 @@ $ ziglocal --commit 0b744da84
 
 This step will take a while. You will find the compiler and library of your local debug build in the `out/zig/build/stage3` directory.
 
+## QA
+
+### Which Zig version do you support?
+
+The Zig toolchain, including the compiler, build system and standard library is still in development and breaking changes do happen every now and then. For this reason this project follows the [Zig master branch](https://github.com/ziglang/zig).
+
+The Nix based development shell uses [zig-overlay](https://github.com/mitchellh/zig-overlay) in conjunction with the `flake.lock` file to pin the zig toolchain version to a recent commit ID.
+
+The dependency is updated by us every now and so often and we try to test and fix breaking changes when updating the toolchain version. We highly recommend to use the projects develoment shell when testing the example extensions provided, otherwise you might have problems to compile the extensions at all.
+
+We understand not everyone is keen to install Nix locally. For getting to know the environment you can build and run a the development shell in a local docker container. Use `./dev/docker/build.sh` to build the container and `./dev/docker/run.sh` to start the dockerized development shell.
+
+The current stable release is verion 0.11. As the build system APIs and package management system are undergoing heavy development recently we chose to stick with the `master` branch for now.
+
+
+### Where is my extension installed?
+
+By default the zig build system installs all artifacts into the local `zig-out` folder.
+
+For example we can see that behavior when building the `char_count_zig` extension:
+
+```
+$ cd examples/char_count_zig
+$ zig build
+$ find zig-out
+zig-out
+zig-out/lib
+zig-out/lib/char_count_zig.dylib
+zig-out/share
+zig-out/share/postgresql
+zig-out/share/postgresql/extension
+zig-out/share/postgresql/extension/char_count_zig.control
+zig-out/share/postgresql/extension/char_count_zig--0.1.sql
+```
+
+If you are not sure wether the build system puts all files into the correct location or in case you generate code it can be helpful to debug your build scripts to install into `zig-out`.
```

Comment:
```suggestion
If you are not sure whether the build system puts all files into the correct location or in case you generate code it can be helpful to debug your build scripts to install into `zig-out`.
```